flink-table/flink-table-planner/pom.xml

```diff
@@ -301,6 +301,11 @@ under the License.
 			<version>${project.version}</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>com.jayway.jsonpath</groupId>
+			<artifactId>json-path</artifactId>
+			<version>2.4.0</version>
+		</dependency>
```

Comment:
You now have to dependencies on json-path (table-planner/-runtime respectively), both defining the version but using the same relocation pattern, which could lead to conflicts at runtime.

You want to remove jayway from the `PlannerModule#KNOWN_MODULE_ASSOCIATIONS` so the planner doesn't have access to this dependency from the table-runtime. 

I would suggest to add a dependencyManagement to `flink-table` to control the version (and then remove the explicit dependency here and just pull it in via calcite).

Reply:
I have removed jayway from the `PlannerModule#KNOWN_MODULE_ASSOCIATIONS` and removed the explicit dependency in the planner, and added a dependency entry in `flink-table`